source/style-guide/style/links-and-cross-references.txt

```diff
@@ -75,38 +75,35 @@ about MongoDB concepts can click through and explore the linked content.
 Avoid Inline References in Tutorials
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Avoid inline textual references in tutorials or stepped procedures. If
-users follow inline references while in the middle of a procedure, they
-may have difficulty navigating back and reorienting themselves with
-their current step.
+- Don't add inline textual references in tutorials or stepped procedures. 
 
-Additionally, avoid stacking multiple admonitions directly after one
-another, as these can be distracting for users. To avoid stacking
-admonitions, consider grouping supplemental information from admonitions
-into a dedicated section.
+  If users follow inline references while in the middle of a procedure, this complicates navigating back and reorienting themselves with
+  their current step.
 
-Inline textual references may be appropriate for more reference-heavy
-pages. Admonitions can potentially reduce scannability of tables and
-options lists, and should therefore be avoided in those contexts
-whenever possible.
+  Inline textual references may be appropriate for more reference-heavy pages,
+  or tasks that aren't part of a streamlined tutorial for a particular action.
+
+- Don't stack multiple admonitions directly after one another, as these
+  can be distracting for users. To avoid stacking admonitions, group
+  supplemental information from admonitions into a dedicated section.
+
+- Don't use admonitions in tables and option lists, if possible, because
+  admonitions reduce scannability of tables and options lists.
 
 Minimize Repeated Links
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Don't link to the same target multiple times in the same section.
+- Don't link to the same target multiple times in the same section.
 
-You may link to the same target multiple times on the same page, but
-think carefully about whether it's absolutely necessary. For example, if
-a page is particularly long and contains many sections, then it might be
-helpful to link to a target an additional time or two. Users may
-immediately navigate to a section far down on a page and miss the
-initial reference.
+  If a page is long and contains many sections, you may link to a target
+  once or twice from this page. This helps users who navigate to a section
+  far down on a page and may miss the initial reference at the top of the
+  page. However, don't link to a target more than twice on the same page.
```

Comment:
I think this is one example scenario, but might not be the only scenario where this would be appropriate.

```suggestion
  You may link to the same target multiple times on the same page, but
  think carefully about whether it's absolutely necessary. For example, if
  a page is particularly long and contains many sections, then it might be
  helpful to link to a target an additional time or two. Users may
  immediately navigate to a section far down on a page and miss the
  initial reference.
```